lib/srv/usermgmt_test.go

```diff
@@ -430,3 +440,44 @@ func TestUpdateUserGroups(t *testing.T) {
 	assert.Equal(t, 1, backend.setUserGroupsCalls)
 	assert.ElementsMatch(t, userinfo.Groups, backend.users["alice"])
 }
+
+func Test_DropExistingUser(t *testing.T) {
```

Comment:
```suggestion
func Test_DontDropExistingUser(t *testing.T) {
```

or include a comment that further explains what behaviour we are asserting for here.